apple/internal/resource_actions/plist.bzl

```diff
@@ -341,19 +341,21 @@ def merge_root_infoplists(
     else:
         plist_key = "MinimumOSVersion"
 
-    input_files.append(environment_plist)
+    if environment_plist:
+        input_files.append(environment_plist)
+        forced_plists.append(environment_plist.path)
```

Comment:
These aren't created for the `macos` platform (e.g. `environment_plist` is `None` here)